overlay/base/usr/lib/system/_os/cli/checkupdates.py

```
@@ -46,14 +55,17 @@ def command(args: Namespace) -> None:
 
     print("\n".join(updates))
     if cast(bool, args.download):
-        image = baseImage()
-        try:
-            if [x for x in updates if x.startswith(f"{image} ")]:
+        if (
+            [x for x in updates if x.startswith(f"{baseImage()} ")]
+            and force
```

Ruff (0.15.9)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)